modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/workflow/processors/SW360EnricherImpl.java

```diff
@@ -54,13 +60,29 @@ public SW360EnricherImpl(IProcessingReporter reporter, SW360MetaDataReceiver con
                 addClearingStateIfAvailable(artifact, sw360Release);
                 mapExternalIdsOnSW360Release(sw360Release, artifact);
                 updateLicenses(artifact, sw360Release);
+
+                if (downloadAttachments) {
+                    downloadAttachments(sw360Release, artifact);
+                }
             } else {
                 warnAndReport(artifact, "No SW360 release found for artifact.", MessageType.PROCESSING_FAILURE);
             }
         }
         return intermediates;
     }
 
+    private void downloadAttachments(SW360Release sw360Release, Artifact artifact) {
```

Comment:
Please do not do this, download only attachments needed for the antenna process

Reply:
I did this with the following thought:
- we want to eventually have attachments for the source, the binary and the clearing state. this totals to 3 kind of attachments we can map on the artifacts. 
- if you check the content of the method it shows that per attachment I call a function `mapAttachmentOnArtifact()`. This checks if the attachment has a type that is something we can map on an artifact and *only then* does it actually download and map the attachment. this is currently just "SOURCE" since the binary and clearing state part is still missing. 
- I can rename the method to something like  `checkAndDownloadAttachments` or something. 

Conclusion: I am not downloading all attachments